packages/netlify-cms-core/src/components/MediaLibrary/MediaLibrary.js

```diff
@@ -350,13 +351,15 @@ class MediaLibrary extends React.Component {
         displayURLs={displayURLs}
         loadDisplayURL={this.loadDisplayURL}
         t={t}
+        mediaFolder={mediaFolder}
       />
     );
   }
 }
 
 function mapStateToProps(state) {
   const { mediaLibrary } = state;
+  const mediaFolder = state.config.media_folder;
```

Comment:
Thanks @DenysLins, this might be accurate as media folders can be configured per collection and per field.

We might pass the untrimmed value from this function https://github.com/netlify/netlify-cms/blob/b94d5f6e7fe07a2816098b377b30a4ddd39c9520/packages/netlify-cms-core/src/reducers/entries.ts#L754

Reply:
OK @erezrokah ... I will apply your suggestions...